React/Fabric/Mounting/ComponentViews/Slider/RCTSliderComponentView.mm

```diff
@@ -62,7 +64,12 @@ - (instancetype)initWithFrame:(CGRect)frame
     _maximumTrackImageResponseObserverProxy = RCTImageResponseObserverProxy(self);
     _thumbImageResponseObserverProxy = RCTImageResponseObserverProxy(self);
 
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
     self.contentView = _sliderView;
+#else
+//    self.contentView = [RCTUISlider new];
+    [self.contentView addSubview:_sliderView];    
```

Comment:
Commented out code? Also is setting to `contentView` directly something we can add to the shim? I'm not sure why we need the extra `addSubview` call from just reading.

Reply:
I reverted this so we set the `_sliderView` as `contentView` directly.